TYPES.md

```diff
@@ -19,6 +48,22 @@ import factory from 'k-redux-factory'
 export default factory({ type: 'keyValue', /* other options */ })
 ```
 
+There are multiple factories signatures, take you preferred between :
+ - `keyValue(middlewares)(options)` : this is a `keyValue`  factory with middlewares
+ - `keyValue(options)` : this is a `keyValue`  factory without middlewares
+
+Parameters are :
+ - **middlewares** (optional), contain an object with `pre` and `post` fields. Both are an array of middlewares to apply before and after the `core` middleware
+ - **options** (mandatory), either a string representating the reducer `name`, either an object with these fields :
+   - **key** (optional), the field used to identify your objects (`id` for example)
```

Comment:
this is not mandatory for `keyValues` ? what is the default key name if nothing is provided ?

Reply:
You are right, there is not default value anymore.